src/octree/mod.rs

```diff
@@ -31,7 +31,7 @@ mod node;
 pub use self::node::{ChildIndex, Node, NodeId, NodeIterator, NodeMeta, NodeWriter,
                      PositionEncoding};
 
-pub const CURRENT_VERSION: i32 = 9;
+pub const CURRENT_VERSION: i32 = 10;
```

Comment:
no need to increase the version? The old protos will load fine, they will just contain no longer used data.